### ISSUE TITLE
error send dash

### ISSUE THREAD

**Issue by umdalidulidali** [akhavr/electrum-dash#301]
plugin object has no attribute get_trezor_output_script_type

https://prnt.sc/23fmjq1

what can I do ?

**Comment by zebra-lucky** [comment]
Do you have Trezor or Safe-T hardware wallet?

**Comment by umdalidulidali** [comment]
> Do you have Trezor or Safe-T hardware wallet?

safe-t mini

**Comment by zebra-lucky** [comment]
This problem seems already fixed from 3.3.4 and in current 4.1.5.0 version.

**Comment by umdalidulidali** [comment]
> 

version dash electrum?

**Comment by zebra-lucky** [comment]
Yes, this was fixed in upstream code: https://github.com/spesmilo/electrum/issues/4726

And integrated in Dash Electrum code from 3.3.4

**Comment by umdalidulidali** [comment]
> Yes, this was fixed in upstream code: [spesmilo/electrum#4726](https://github.com/spesmilo/electrum/issues/4726)
> 
> And integrated in Dash Electrum code from version 3.3.4

great! thx!
